### PR TITLE
update icon: denojs (orginial, original-wordmark)

### PR DESCRIPTION
**Pull request by PantheraRed**
## Double check these details before you open a PR

- [x] PR does not match another non-stale PR currently opened
- [ ] PR name matches the format *new icon: <i>Icon name</i> (<i>versions separated by comma</i>)*. More details [here](https://github.com/devicons/devicon/blob/develop/CONTRIBUTING.md#overview)
- [x] PR's base is the `develop` branch.
- [x] Your icons are inside a folder as seen [here](https://github.com/devicons/devicon/blob/develop/CONTRIBUTING.md#organizational-guidelines)
- [x] SVG matches the standards laid out [here](https://github.com/devicons/devicon/blob/develop/CONTRIBUTING.md#svgStandards)
- [ ] A new object is added in the `devicon.json` file as seen [here](https://github.com/devicons/devicon/blob/develop/CONTRIBUTING.md#-updating-the-deviconjson-)

**This PR closes NONE**

## Link to prove your SVG is correct and up-to-date.

https://deno.land/
https://github.com/denoland/deno

### CODE REVIEW

**Comment by PantheraRed** [comment]
Proposal to change `denojs` to `deno` since its official name is `deno` and not `denojs` (denojs can be its `altname`).

**Comment by Snailedlt** [comment]
> Proposal to change `denojs` to `deno` since its official name is `deno` and not `denojs` (denojs can be its `altname`).

I agree with the change, but it's a breaking change, so we will have to wait for the next major release for this one. Please make an issue about it though, so we can track it :)

**Review by Snailedlt — APPROVED** [review]
LGTM! :heavy_check_mark: 
Thank you for your contribution! :100:

File: devicon.json [hunk 1]
```diff
@@ -1984,7 +1984,8 @@
         "tags": [
             "javascript",
             "rust",
-            "language"
+            "language",
```
Comment (lunatic-fox) [inline devicon.json:1987]: Should we be keeping this tag here? 🤔
I think it fits more when the technology is actually a language, like JavaScript or Rust, but not when it uses some language like this one, React or some other Framework.
Let me know what you guys think. 🙂
```suggestion
            "language",
```

Comment (PantheraRed) [reply devicon.json:1987]: I agree. The `language` tag doesn't quite fit `deno` and `nodejs`.

**Review by Snailedlt — APPROVED** [review]

**Review by lunatic-fox — APPROVED** [review]
Approved! ✔
Thank you for this contribution! 🚀

**Pull request merged**